network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/styleguide_spacing_table.html

```diff
@@ -0,0 +1,195 @@
+                </tr>
+                <tr class="tw-bg-black tw-bg-opacity-5">
+                    <td class="tw-p-6 tw-border tw-border-table tw-font-bold">88</td>
+                    <td class="tw-p-6 tw-border tw-border-table">176px (11rem)</td>
+                    <td class="tw-p-6 tw-border tw-border-table"><div class="tw-bg-red-40 tw-w-88 tw-h-88"></div></td>
+                </tr>
+                <tr class="">
+                    <td class="tw-p-6 tw-border tw-border-table tw-font-bold">96</td>
+                    <td class="tw-p-6 tw-border tw-border-table">192px (12rem)</td>
+                    <td class="tw-p-6 tw-border tw-border-table"><div class="tw-bg-red-40 tw-w-96 tw-h-96"></div></td>
+                </tr>
+                <tr class="tw-bg-black tw-bg-opacity-5">
+                    <td class="tw-p-6 tw-border tw-border-table tw-font-bold">104</td>
+                    <td class="tw-p-6 tw-border tw-border-table">208px (13rem)</td>
+                    <td class="tw-p-6 tw-border tw-border-table"><div class="tw-bg-red-40 tw-w-104 tw-h-104"></div></td>
+                </tr>
+                <tr class="">
+                    <td class="tw-p-6 tw-border tw-border-table tw-font-bold">112</td>
+                    <td class="tw-p-6 tw-border tw-border-table">224px (14rem)</td>
+                    <td class="tw-p-6 tw-border tw-border-table"><div class="tw-bg-red-40 tw-w-112 tw-h-112"></div></td>
+                </tr>
+                <tr class="tw-bg-black tw-bg-opacity-5">
+                    <td class="tw-p-6 tw-border tw-border-table tw-font-bold">120</td>
+                    <td class="tw-p-6 tw-border tw-border-table">240px (15rem)</td>
+                    <td class="tw-p-6 tw-border tw-border-table"><div class="tw-bg-red-40 tw-w-120 tw-h-120"></div></td>
+                </tr>
+                <tr class="">
+                    <td class="tw-p-6 tw-border tw-border-table tw-font-bold">128</td>
+                    <td class="tw-p-6 tw-border tw-border-table">256px (16rem)</td>
+                    <td class="tw-p-6 tw-border tw-border-table"><div class="tw-bg-red-40 tw-w-128 tw-h-128"></div></td>
+                </tr>
+                <tr class="tw-bg-black tw-bg-opacity-5">
+                    <td class="tw-p-6 tw-border tw-border-table tw-font-bold">144</td>
+                    <td class="tw-p-6 tw-border tw-border-table">288px (18rem)</td>
+                    <td class="tw-p-6 tw-border tw-border-table"><div class="tw-bg-red-40 tw-w-144 tw-h-144"></div></td>
+                </tr>
+                <tr class="">
+                    <td class="tw-p-6 tw-border tw-border-table tw-font-bold">160</td>
+                    <td class="tw-p-6 tw-border tw-border-table">320px (20rem)</td>
+                    <td class="tw-p-6 tw-border tw-border-table"><div class="tw-bg-red-40 tw-w-160 tw-h-160"></div></td>
+                </tr>
```

Comment:
Can we add a row for `192` as well? Thanks!

Reply:
http://localhost:8000/en/style-guide/#spacing is updated now